src/_locales/en/messages.json

```diff
@@ -1,142 +1,166 @@
+	},
+	"notOnFacebookHeader": {
+		"message": "$URL$ is not in the Facebook Container.",
+		"description": "This is shown at the top of the panel pop-up when the user is NOT on a Facebook website, which has been automatically opened OUTSIDE Facebook Container. DO NOT TRANSLATE \"Facebook Container\".",
+		"placeholders": {
+			"url": {
+				"content": "$1",
+				"example": "www.google.com"
+			}
+		}
+	},
+	"notOnFacebookText": {
+		"message": "This site is outside of the Facebook Container. This makes it harder for Facebook to track you around the web.",
+		"description": "This is shown in the body of the panel pop-up when the user is NOT on a Facebook website, which has been automatically opened OUTSIDE Facebook Container. DO NOT TRANSLATE \"Facebook Container\"."
+	},
+	"onUnknownSiteHeader": {
+		"message": "This site is not in the Facebook Container.",
+		"description": "This is shown at the top of the panel pop-up when the user is on an \"unknown\" website - e.g., about:newtab. DO NOT TRANSLATE \"Facebook Container\"."
+	},
+	"facebookContainer": {
+		"message": "Facebook Container",
+		"description": "This is the name of the extension, and appears at the top of all onboarding panels. Please do not translate \"Facebook Container\"."
+	},
+	"onboarding1-subhead": {
+		"message": "Using Facebook Container",
+		"description": "This is the subhead of onboarding panel 1. Please do not translate \"Facebook Container\""
+	},
+	"onboarding1-p1": {
+		"message": "Facebook can track your browsing far beyond Facebook.com. Facebook Like and Share buttons, for example, can track where you’ve been — *SPANSTART even on pages you never like or share. *SPANEND",
+		"description": "This is the first paragraph of onboarding panel 1. Please do not translate \"Facebook Container\""
+	},
+	"onboarding1-p2": {
+		"message": "Facebook Container creates a boundary between Facebook sites and the rest of the web so it’s harder for Facebook to track you online.",
+		"description": "Please do not translate \"Facebook Container\""
+	},
+	"onboarding2-subhead": {
+		"message": "On Facebook and Sites in the Facebook Container Boundary",
+		"description": "This is the subhead for the second onboarding panel. Please do not translate \"Faceook Container\""
+	},
+	"onboarding2-alt": {
+		"message": "The Facebook Container fence icon",
+		"description": "This is alternative text for the icon in the second onboarding panel"
+	},
+	"onboarding2-p1": {
+		"message": "This icon appears in the Firefox address bar when you visit sites, like Facebook.com, that are in the Facebook Container boundary.",
+		"description": "This is the first paragraph of the second onboarding panel."
+	},
+	"onboarding2-p2": {
+		"message": "These sites need to access to your Facebook identity (or know who you are) in order to work. *SPANSTART This also means that Facebook can track everything you do on these sites. *SPANEND",
```

Comment:
Just a small typo; I believe it should be just "to access" instead of "to access to".

Reply:
Revised!